src/mlpack/methods/ann/layer/adaptive_max_pooling.hpp

```diff
@@ -51,7 +51,14 @@ class AdaptiveMaxPooling
    * @param outputShape A two-value tuple indicating width and height of the output.
    */
   AdaptiveMaxPooling(const std::tuple<size_t, size_t>& outputShape);
-
+  // Copy Constructor
+  AdaptiveMaxPooling(const AdaptiveMaxPooling& layer);
+  // Move Constructor
+  AdaptiveMaxPooling(AdaptiveMaxPooling&&);
+  // Copy assignment operator
+  AdaptiveMaxPooling& operator=(const AdaptivrMaxPooling& layer);
```

Comment:
```suggestion
  AdaptiveMaxPooling& operator=(const AdaptiveMaxPooling& layer);
```

Looks like this is a typo.

Reply:
Hey @Abilityguy Thanks for pointing out typos, I also wrote tests for the same, this is my first time writing tests, can you go through it and make sure it's correct?